themes/default/content/blog/deploy-aws-static-website-with-angular/index.md

```diff
@@ -0,0 +1,116 @@
+
+A simple website, like the one we’ll be building in this post, is mainly created from a group of Angular components.
+
+## What is the AWS Static Website Component?
+
+The AWS Static Website component from Pulumi makes it easy to deploy an AWS S3 static website and, optionally, add a CloudFront content distribution network (CDN). While you can use any of the programming languages Pulumi supports (TypeScript, JavaScript, Python, Go, .NET, Java, and YAML), we’ll be using TypeScript, since that is what the Angular framework uses.
+
+The great advantage of the AWS Static Website component is that you can have a complete, functioning site in a few minutes. The component encapsulates all the complex logic that an AWS static website requires. Without the component, you would need to spend hours or even days writing and debugging code to get the same result.
+
+If you’re interested in learning more about the component, read [Deploy Static Websites to AWS using 10 lines of YAML]({{< relref "/blog/deploy-aws-static-website-with-yaml" >}}).
+
+## The Angular Components
+
+All the Angular components you need to build the website are available in the GitHub [repository](https://github.com/sean1588/angular-static-site-pulumi). The repository contains:
+
+* A component for the [left navigation bar](https://github.com/sean1588/angular-static-site-pulumi/tree/main/src/app/components/shared/leftnav).
+* A component for the [top navigation bar](https://github.com/sean1588/angular-static-site-pulumi/tree/main/src/app/components/shared/topnav).
+* A component for the [Home](https://github.com/sean1588/angular-static-site-pulumi/tree/main/src/app/components/home) page.
+* A component for the [Menu](https://github.com/sean1588/angular-static-site-pulumi/tree/main/src/app/components/menu) page.
+
+We also use some components that come with the Angular framework.
+
+Each of the components has a TypeScript file that contains the programming logic and an HTML file that describes how the component will render.
+
+## Building the Pulumi Café Website
+
+Here are the instructions to build the website.
+
+1. Clone the GitHub repository located [here](https://github.com/sean1588/angular-static-site-pulumi).
+1. Run `npm install` to install the website’s node dependencies.
+1. Run `npm build` to generate the static website files and place them in a directory called **dist**. We will reference this directory when we configure Pulumi to deploy the site.
+
+## Deploy the Site with Pulumi
+
+You’re now ready to set up Pulumi to deploy the site to AWS.
+
+1. Create a directory named **pulumi** in the root of the project directory and then navigate to it with this command:
+
+    ```bash
+    mkdir pulumi && cd pulumi
+
+    ```
+
+1. You’re now in the **pulumi** directory. Generate a Pulumi project file and create a Pulumi.yaml file with this command:
+
+    ```bash
+    pulumi new aws-yaml
+    ```
+
```

Comment:
We should probably mention that the project should be named `angular-static-website`, just to match what's in the example code. I'll usually say something along the lines of "follow the prompts" as well just so they know that's coming.

Reply:
sounds good. I'll add a note there for that.